From d767344a131b1485a16bc8e1526bf0f2036d6052 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Tue, 15 Oct 2019 18:07:52 +0800
Subject: [PATCH] power: supply: rk818_charger: fix gcc warning

drivers/power/supply/rk818_charger.c: In function 'rk818_cg_usb_get_property':
drivers/power/supply/rk818_charger.c:448:11: warning: 'fake_offline' may be used uninitialized in this function [-Wmaybe-uninitialized]
   else if (fake_offline)
           ^

Change-Id: I07ccf6e136d47c75db56004de1c78436e570a356
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/power/supply/rk818_charger.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/power/supply/rk818_charger.c b/drivers/power/supply/rk818_charger.c
index 35f08ff2d1d3..c7f7211c0138 100644
--- a/drivers/power/supply/rk818_charger.c
+++ b/drivers/power/supply/rk818_charger.c
@@ -426,7 +426,7 @@ static int rk818_cg_usb_get_property(struct power_supply *psy,
 				     union power_supply_propval *val)
 {
 	struct rk818_charger *cg = power_supply_get_drvdata(psy);
-	int fake_offline, ret = 0;
+	int fake_offline = 0, ret = 0;
 
 	if (rk818_cg_online(cg))
 		fake_offline = rk818_cg_lowpwr_check(cg);
-- 
2.35.3

